src/daft-scan/src/glob.rs

```diff
@@ -272,9 +272,28 @@ impl ScanOperator for GlobScanOperator {
     }
 
     fn multiline_display(&self) -> Vec<String> {
+        let condensed_glob_paths = if self.glob_paths.len() <= 6 {
```

Comment:
This should be `7`, otherwise we take the same amount of lines and omit 1 file.

Reply:
done